src/features/common/LiveRegion/LiveRegion.module.scss

```
@@ -0,0 +1,15 @@
+// Screen-reader-only content: removed from the visual flow so it reserves no
+// space, while remaining announced by assistive tech.
+// `display: none` / `visibility: hidden` would remove it from the a11y tree,
+// so the clip technique is used instead.
+.visuallyHidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)